src/test/HST/Util/FreeVarsTests.hs

```diff
@@ -0,0 +1,70 @@
+-- | This module contains tests for "HST.Util.FreeVars".
+module HST.Util.FreeVarsTests where
+
+import           Polysemy                  ( Members, Sem )
+import           Test.Hspec
+  ( Spec, context, describe, it, shouldBe )
+
+import           HST.Effect.Cancel         ( Cancel )
+import           HST.Effect.Report         ( Report )
+import           HST.Effect.SetExpectation ( SetExpectation, setExpectation )
+import           HST.Effect.WithFrontend   ( WithFrontend )
+import           HST.Frontend.Syntax       as S
+import           HST.Test.Parser           ( parseTestModule )
+import           HST.Test.Runner           ( runTest )
+import           HST.Util.FreeVars         ( freeVars )
+
+-------------------------------------------------------------------------------
+-- Expectation Setters                                                       --
+-------------------------------------------------------------------------------
+-- | Parses the given Haskell module and sets the expectation that the given
+--   variables are free.
+--
+--   All free
```

Comment:
Incomplete sentence.

Reply:
I have no idea what I wanted to write. Removed the paragraph in 193b7032af23730cd243db08b2a242814278727f.